community-content/pytorch_pre_built_images_deployment/pytorch-text-classification-vertex-ai-deploy.ipynb

```diff
@@ -0,0 +1,1658 @@
+{
```

Comment:
<p>Can we use this language:</p><p> </p><p><strong>This is an Experimental release</strong>, covered by the Pre-GA Offerings Terms of your Google Cloud Platform <a href="https://cloud.google.com/terms" target="_blank">Terms of Service</a>.</p><p> </p><p>Experiments are focused on validating a prototype and are not guaranteed to be released. They are not intended for production use or covered by any SLA, support obligation, or deprecation policy and might be subject to backward-incompatible changes.</p><p> </p><p><strong>Kindly drop us a note before you run any scale tests.</strong></p><p> </p><p><strong>Do not hesitate to contact </strong><a href="mailto:cloudml-feedback@google.com" target="_blank">cloudml-feedback@google.com</a><strong> if you have any questions or run into any issues.</strong></p><p> </p><p>The usage of the product is free during the Experimental release period: you will still incur charges for other GCP products usage, such as storage. </p><p> </p><p>(And then add the details on allowlisting. Q - do customers need to share project ID or project number  ?)</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Either project id or project number is fine.